pyproject.toml

```diff
@@ -44,11 +44,13 @@ markdown = { version = "^3.3.3", optional = true }
 pytest = { version = "^6.0.0", optional = true }
 pytest-check = { version = "^1.0.1", optional = true }
 pytest-cov = { version = "^2.5", optional = true }
+mypy = {version = "^0.961", optional = true}
```

Comment:
Total nit here, if it's easy could you actually put this in alphabetical order? And below. But if not easy don't worry about it